src/info/title.rs

```diff
@@ -1,83 +1,155 @@
-use super::get_style;

```

Comment:
:thinking: Looks like this file was (correctly) converted to LF line endings.
For the sake of a reasonable diff, I think we should convert all files to LF so that it's easier to compare this and https://github.com/o2sh/onefetch/pull/813

Reply:
I saw the CR LF in title.rs, was confused and ran dos2unix on it, but then had a bunch of diffs for this change so continued with the "match existing" attittude.

Respond one more time to confirm ok to convert the line endings to LF and I will do so.

Reply:
>Oops, did I do that?

You *correctly* did that (See `.editorconfig` file) :laughing: 

I *incorrectly* set up an EditorConfig and never enforced its settings in a workflow.